### ISSUE TITLE
Please update to GVR SDK v1.110.0

### ISSUE THREAD

**Issue by ylluminarious** [googlevr/daydream-elements#7]
I'm trying to add Daydream Elements to my Unity project and I'm seeing a lot of conflicts in the Unity import window. A lot of these conflicts are  just version differences in the GVR SDK parts. Please update Daydream Elements to the newest version of the SDK. That would help lessen the clutter for others like myself.

Also, maybe you guys should automate the process of updating the SDK in this repository. That would probably be easier than updating by hand for each new release of the SDK. Just my 2¢

**Comment by ylluminarious** [comment]
To anyone else suffering from this issue, I worked around the problem by:

1. deleting `Assets/GoogleVR`
2. re-importing an older version of the SDK (v1.100.1, which is the most recent version that is compatible with Daydream Elements)
3. finally I applied the most recent version of the SDK on top of that (v1.110.0).

This *does* feel hacky and may be subject to certain things not being updated correctly, but I have not encountered any such issues so far and it seems to be the most viable solution for now. Of course, I would recommend keeping everything under version control so that you don't inadvertently screw up your project.
